guide/using-python.md

```diff
@@ -17,22 +17,18 @@ On Ubuntu 16.04, it also supports Python 3.5 and on Ubuntu 14.04, it also suppor
 
 ### Installation
 
+Webots will start Python using the standard `python` command line. As a consequence, it will execute the first `python` binary found in the current `PATH`. If you need a different version of Python, please install it if needed and configure your environment so that it becomes the default `python` version when called from the command line in a terminal. Alternatively, you can change the default Python command from the Webots Preferences in the General tab. If you set it for example to `python3.6` instead of `python`, this version of python will be used (if available from the command line). Finally, it is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.6` or `python2.7`, etc. If specified in the `runtime.ini` file of a controller, this Python command will be executed instead of the default one to launch this controller.
+
 #### macOS and Linux installation
 
-Most of the Linux distributions have Python 2.7 and 3.x already installed.
-macOS has Python 2.7 installed by default.
+Most of the Linux distributions have Python 2.7 and 3.x already installed. macOS has Python 2.7 installed by default.
```

Comment:
Personally I think that it is much better to keep sentence per line as it was before.